cluster/operations/bbr.yml

```diff
@@ -0,0 +1,26 @@
+- type: replace
+  path: /releases/-
+  value:
+    name: backup-and-restore-sdk
+    url: https://bosh.io/d/github.com/cloudfoundry-incubator/backup-and-restore-sdk-release?v=((bbr_sdk_version))
+    sha1: ((bbr_sdk_sha1))
+    version: ((bbr_sdk_version))
+
+- type: replace
+  path: /instance_groups/name=web/jobs/-
+  value:
+    name: bbr-atcdb
+    properties:
+      postgresql:
+        database: atc
+        role:
+          name: concourse
```

Comment:
Just like in the other comment, this *might* be important to be parametrizable. Wdyt?

thx!